src/Bonsai.ML.Visualizers/KinematicStateVisualizer.cs

```diff
@@ -0,0 +1,179 @@
+using Bonsai.Design;
+using Bonsai;
+using Bonsai.ML.LinearDynamicalSystems.Kinematics;
+using Bonsai.ML.Visualizers;
+using System.Collections.Generic;
+using System;
+using System.Windows.Forms;
+using System.Reactive.Linq;
+using System.Linq;
+using System.Reactive;
+
+[assembly: TypeVisualizer(typeof(KinematicStateVisualizer), Target = typeof(KinematicState))]
+
+namespace Bonsai.ML.Visualizers
+{
+
+    /// <summary>
+    /// Provides a type visualizer to display the state components of a Kalman Filter Kinematics model.
+    /// </summary>
+    public class KinematicStateVisualizer : MashupVisualizer
+    {
+        internal int RowCount { get; set; } = 3;
+        internal int ColumnCount { get; set; } = 2;
+        internal List<StateComponentVisualizer> componentVisualizers { get; private set; } = new();
```

Comment:
Just one final question: is there a particular reason to have these be internal properties rather than private fields? They don't seem to be used anywhere in this PR, but maybe the plan is they will be used in subsequent developments?

If they will be used this way, maybe `componentVisualizers` should be cased as `ComponentVisualizers`.